server/src/main/java/org/candlepin/dto/api/v1/DistributorVersionTranslator.java

```diff
@@ -60,28 +66,43 @@ public DistributorVersionDTO populate(DistributorVersion source, DistributorVers
     public DistributorVersionDTO populate(ModelTranslator modelTranslator, DistributorVersion source,
         DistributorVersionDTO dest) {
 
-        dest = super.populate(modelTranslator, source, dest);
+        if (source == null) {
+            throw new IllegalArgumentException("source is null");
+        }
 
-        dest.setId(source.getId());
-        dest.setName(source.getName());
-        dest.setDisplayName(source.getDisplayName());
+        if (dest == null) {
+            throw new IllegalArgumentException("dest is null");
+        }
 
-        if (modelTranslator != null) {
+        dest.id(source.getId())
+            .name(source.getName())
+            .displayName(source.getDisplayName())
+            .created(toDateTime(source::getCreated))
+            .updated(toDateTime(source::getUpdated));
 
-            Set<DistributorVersionCapability> capabilities = source.getCapabilities();
-            if (capabilities != null && !capabilities.isEmpty()) {
-                for (DistributorVersionCapability capability : capabilities) {
-                    if (capability != null) {
-                        dest.addCapability(new DistributorVersionDTO.DistributorVersionCapabilityDTO(
-                            capability.getId(), capability.getName()));
-                    }
-                }
-            }
-            else {
-                dest.setCapabilities(Collections.emptySet());
+        if (modelTranslator != null) {
```

Comment:
@Januson 
`modelTranslator` never used in this code. Any specific reason for this null check?

Reply:
No reason. Just forgot to remove it. Done.